types/mclimate/vicki/mclimate.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+    "$id": "https://akenza.io/mclimate/vicki/mclimate.schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "MClimate",
+    "type": "object",
+    "properties": {
+      "deviceId": {
+        "title": "Device EUI",
+        "description": "The device EUI",
+        "type": "string"
```

Comment:
Hoi @tintsch please add `"hideFromKpis":true` for all those properties so they do not get shown in the overview page